tests/test_metadata_tools.py

```diff
@@ -1,13 +1,14 @@
 import unittest
-
+from gen_import_utils import read_json_config
 import pandas as pd
 import shutil
 from metadata_tools import MetadataTools
 class TestMetadataTools(unittest.TestCase):
 
     def setUp(self):
         self.path = "tests/test_images/test_image.jpg"
```

Comment:
needs a default config file for the config argument,  the config file changing won't effect the individual tests, which are made using prepared data in setup.